src/components/Quiz.js

```diff
@@ -30,15 +31,28 @@ const Quiz = () => {
     <Container>
       <h1>CYF Quiz</h1>
       <Form onSubmit={checkUserAnswers}>
-        {data.map(question => {
-          return (
-            <Question
-              key={question.id}
-              question={question}
-              handleAnswerSelect={handleAnswerSelect}
-            />
-          );
-        })}
+        <Question
+          key={data[currentQuestionIndex].id}
+          question={data[currentQuestionIndex]}
+          handleAnswerSelect={handleAnswerSelect}
+          selectedAnswer={userAnswers[data[currentQuestionIndex].id]}
+        />
+        <Button
+          className="mr-2 mb-4"
+          variant="secondary"
+          disabled={currentQuestionIndex === 0}
+          onClick={() => setCurrentQuestionIndex(currentQuestionIndex - 1)}
+        >
+          Previous
+        </Button>
+        <Button
+          className="mr-2 mb-4"
+          variant="secondary"
```

Comment:
It's common practise in web development to have a primary button that you want most users to click. For example, when you have a `Cancel` and a `Submit` button, you want the user to click `Submit` rather than `Cancel`, so we do that by making the `Submit` button more grey and the `Submit` button more colourful. Here, we could make the `variant` `primary` since we want the users to be clicking `Next`

Reply:
changed